actions/setup-conftest/action.yaml

```diff
@@ -18,6 +18,14 @@ runs:
         path: ${{ github.workspace }}/bin/conftest
         key: conftest-${{ runner.os }}-${{ runner.arch }}-${{ inputs.version }}
 
+    - name: Normalize runner.os to match Conftest release
+      shell: sh
+      env:
+        OS: ${{ runner.os }}
+      run: |
+        [[ "$OS" == 'macOs' ]] && OS='Darwin'
```

Comment:
btw the [docs](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#runner-context) say `macOS` with an upper case `S`. they could be wrong, just pointing it out in case this is a typo